### ISSUE TITLE
Removes words wrapped in <tags> from heading

### ISSUE THREAD

**Issue by iamchriswick** [xuekai-china/vuepress-plugin-right-anchor#11]
The plugin do not show words wrapped in i.e. `<span>` and `<strong>` tags. Can you please fix this?

**Comment by xuek9900** [comment]
`<h1>` - `<h6>` tags in `<span>` or `<strong>` tags?

**Comment by iamchriswick** [comment]
@xuekai-china The content in `<span>` and `<strong>` tags inside `<h1>` - `<h6>`.

This is my header:
```
<h3><a href="#" class="header-anchor">#</a> <span class="badge get">Lorem</span> /payment_request/<code>{ tid }</code>/outcome/</h3>
```
In the right anchorbar it removes whatever text is between `<span class="badge get">` and `</span`

**Comment by xuek9900** [comment]
OK, I'll see why.

**Comment by xuek9900** [comment]
I tested it on a project I was developing, not use `right-anchor`，so I only tested it from the data layer.
As shown below，I tried two models and they got different results.
<br>
![image](https://user-images.githubusercontent.com/49433924/110882433-dde49480-831c-11eb-9f9f-e1078dffb01c.png)
![image](https://user-images.githubusercontent.com/49433924/110882581-2734e400-831d-11eb-8c60-db5b0b270098.png)
<br>
So I don't think it's `ight-anchor` problem. It's `vuepress` rules when parsing headers.
